dev-docs/source/Eclipse.rst

```diff
@@ -0,0 +1,121 @@
+.. _Eclipse:
+
+=================
+Eclipse on Ubuntu
+=================
+
+.. contents::
+	:local:
+	
+Prerequisites
+################
+1.
+Follow the getting started guide and building with cmake, but stop before actually using cmake to build the project.
+
+2.
+Install the JRE by running sudo apt-get install default-jre
+
+	
+Installing Eclipse
+###################
+1.
+Install Ubuntu-Make
+
+run the following in terminal
+
+.. code-block:: sh
+
+  sudo add-apt-repository ppa:ubuntu-desktop/ubuntu-make 
+  sudo apt update
+  sudo apt install ubuntu-make
+  
+	
+This will add the umake repository to the list of repositories then install umake
+
+2.
+Install eclipse using Umake
+
+run the following in terminal
+
+.. code-block:: sh
+
+	umake ide eclipse-cpp
+	
+leave the installation path to the default
+
+3.
+add eclipse to the path so it can be run from command line
+
+run the following in terminal
+
+.. code-block:: sh
+
+	gedit .bashrc 
+	
+then scroll to the bottom of the file and add 
+
+.. code-block:: sh
+
+	export PATH=$PATH:/path/to/eclipse-cpp-folder/
+	
+then save and exit gedit.
+
```

Comment:
I'd recommend telling them to restart the terminal so it loads from the .bashrc file again